crates/turborepo-lib/src/run/mod.rs

```diff
@@ -102,6 +111,7 @@ impl<'a> Run<'a> {
             TurboState::platform_name(),
         );
         let start_at = Local::now();
+        self.connect_process_manager(signal_subscriber);
```

Comment:
In the future this probably belongs in the constructor, but at the moment that's also in the Go codepath when we compare task hashes across Rust/Go and we do not want to setup a signal handler in that case.